readme.md

```diff
@@ -188,6 +183,47 @@ mbo.finalize().then(function(mediaBlobs) {
 });
 ```
 
+### Error Handling in finalize
+When *[finalize()](#finalize-method)* is called, the User Agent will perform these basic checks for the operations that are batched. This error checking should be done before executing any of the operations.
+
+For *[trim()](#trim-method)*
+1. Let *O* represent the blob to be trimmed
+2. If *startTime* is less than 0 **OR** *endTime* is greater than the *O*.duration **OR** *startTime* is greater than the *endTime*:
+    * create a new [ErrorEvent](https://html.spec.whatwg.org/multipage/webappapis.html#errorevent)
+    * report the exception event 
+    * break
+
+For *[split()](#split-method)*
+1. Let *O* represent the blob to be split
+2. If *time* is less than 0 **OR** is greater than *O*.duration **OR** this is not the last operation before finalize() was called
+    * create a new [ErrorEvent](https://html.spec.whatwg.org/multipage/webappapis.html#errorevent)
+    * report the exception event 
+    * break
+
+For *[concat()](#concat-method)*
+1. Let *m1* represent the first *MediaBlob* which will be the *MediaBlob* from the *MediaBlobOperation* that has the *concat* method called upon
+2. Let *m2* represent the *MediaBlob* that is passed in to *concat* method to be concatenated with *m1*
+3. If the mimeType of m1 does not equal the mimeType of m2:
```

Comment:
What if the mimeType of either blob is unknown?  Do we need to call out special behavior for this?

Reply:
If the mimeType is unknown here, we will not even create the MediaBlob.
The MediaBlob constructor will throw an exception if the mimeType is unknown here.